pkg/prompts/prompts.go

```diff
@@ -244,3 +272,40 @@ func Select[T any](label string, items []T, opt *SelectOpt[T]) (T, error) {
 
 	return items[i], nil
 }
+
+func getCurrentDirName() (string, error) {
+	dir, err := os.Getwd()
+	if err != nil {
+		log.Fatalf("getting current directory: %v", err)
```

Comment:
we shouldn't be using fatal here. Use error like the method signature implies.

Reply:
this func should return an error when an error occurs. currently it can't return anything other than nil for the error